packages/modules/pricing/src/services/pricing-module.ts

```diff
@@ -841,6 +838,35 @@ export default class PricingModuleService
     return isString(idOrSelector) ? pricePreferences[0] : pricePreferences
   }
 
+  @InjectTransactionManager("baseRepository_")
+  protected async createPricePreferences_(
+    data: PricingTypes.CreatePricePreferenceDTO[],
+    @MedusaContext() sharedContext: Context = {}
+  ) {
+    const preferences = await this.pricePreferenceService_.create(
+      data.map((d) => ({
+        ...d,
+        is_tax_inclusive: d.is_tax_inclusive ?? false,
```

Comment:
any particular reason why we need this default value here?

Reply:
If someone passes `null` mikroORM throws instead of using the default value, so we need to normalize it

Reply:
interesting 👍🏻